api/cpp/cbindgen.rs

```diff
@@ -692,6 +692,7 @@ fn gen_backend_qt(
         "NativeTabWidget",
         "NativeTab",
         "NativeStyleMetrics",
+        "QtStylePalette",
```

Comment:
Should be called "NativeStylePlalette" for consistency with all the other Qt stuff.

Arguably, we could also decide to rename everything else to `Qt...` instead of `Native...`   But that'd be a different change

Reply:
renamed it. agree with the later renaming.